api/validator.js

```diff
@@ -44,7 +45,8 @@ const SCHEMA_OUTPUT = Joi.object()
     .keys({
         jobs: SCHEMA_JOBS,
         workflow: Workflow.workflow,
-        errors: Joi.array().items(Joi.string()).optional()
+        errors: Joi.array().items(Joi.string()).optional(),
+        annotations: Annotations.annotations
```

Comment:
I don't think this should be here. I think it should be inside `SCHEMA_JOB_PERMUTATION` instead.